lib/features/composer/presentation/widgets/email_address_input_builder.dart

```diff
@@ -180,23 +187,31 @@ class EmailAddressInputBuilder {
               if (_isCollapse) _buildCounter(context, listEmailAddress.length - newListEmailAddress.length),
             ]),
             onTagChanged: (String value) {
-              setState(() => listEmailAddress.add(EmailAddress(null, value)));
-              _onUpdateListEmailAddressAction?.call(_prefixEmailAddress, listEmailAddress);
-            },
-            findSuggestions: (String query) {
-              if (query.trim().isNotEmpty && _onSuggestionEmailAddress != null) {
-                return _onSuggestionEmailAddress!(query.trim());
+              log('EmailAddressInputBuilder::_buildTagEditor(): onTagChanged: $value');
+              if (!_isDuplicatedRecipient(value)) {
+                setState(() => listEmailAddress.add(EmailAddress(null, value)));
+                _onUpdateListEmailAddressAction?.call(_prefixEmailAddress, listEmailAddress);
               }
-              return [];
+              _gapBetweenTagChangedAndFindSuggestion = Timer(
```

Comment:
why need `Timer` at this?

Reply:
Good question.
When you are typing a recipient -> [space] to select -> `tagEditor` will trigger `onTagChanged` and then trigger `findSuggestion` with `recently added email` -> this make the suggestion box show right away. That the reason I want to prevent it by timer.